src/proto/loaded_image.rs

```diff
@@ -17,7 +17,7 @@ pub struct LoadedImage {
 
     // Source location of the image
     device_handle: Handle,
-    _file_path: *const c_void, // TODO: not supported yet
+    file_path: *const c_void,
```

Comment:
The type here can be changed to `*const DevicePath`.

Reply:
It's changed😃